src/parser/parseGeoID.js

```diff
@@ -0,0 +1,122 @@
+const fs = require('fs');
+const utils = require('./utils');
+
+// need to add a display/surpassed parameter in obj
+
+function parseSim(path, series, getIdx, simNums, geoInput, headers, parameters) {
+    // mutates a series (Array of Object) by adding new parsed Simulation File info
+    // parses one simulation of one geoInput
+    try {
+        const data = fs.readFileSync(path, 'UTF-8');
+        const lines = data.split(/\r?\n/);
+        const fileName = path.split('/').splice(-1).pop()
+        const simNum = parseInt(fileName.split('_death-')[1].split('.')[0]);
+
+        // reduce simulations down to 30%
+        if (simNum % 3 === 0) {
+            console.log('sim', simNum)
+            simNums.push(simNum);
+    
+            lines.forEach((line) => {
+                const splitLine = line.split(',')
+                const date = splitLine[0]
+                const geoid = splitLine[headers.indexOf('geoid')];
+    
+                if (geoid === geoInput) {
+                    // skip header and empty lines
+                    if (date.length > 1 && date !== 'time') {
+    
+                        for (let i = 0; i < parameters.length; i ++) {
+                            const stat = parameters[i]; 
+                            const val = parseInt(splitLine[getIdx[stat]]);
+    
+                            if (simNum in series[i][stat]) {
+                                series[i][stat][simNum]['vals'].push(val);
+                            } else {
+                                series[i][stat][simNum] = {'name': simNum, 'vals': [val]};
+                            }
+                        }
+                    }
+                }
+            });
+        }
+    } catch (err) {
+        console.error(err);
+    };
+};
+
+function parseScenario(dir, geoInput, severities, parameters) {
+    // parse entire Scenario Directory
+    const files = fs.readdirSync(dir)
+        .filter(file => file !== '.DS_Store');
+    const dates = utils.getDates(dir + files[0]);
+    const headers = utils.getHeaders(dir + files[0]);
+    const obj = utils.initObj(dates);
+
+    for (let s = 0; s < severities.length; s ++) {
```

Comment:
Unless you need `s` for the count, you can use `for (val severity in severities)` syntax. This allows you to just use `severity` instead of `severity[s]`. 

It can help reduce brackets like in the line
`const statObj = obj[severities[s]][stats[i]];` -> `const statObj = obj[severity][stats[i]];`